src/entities/DealTokenSwap.ts

```diff
@@ -120,6 +120,7 @@ export class DealTokenSwap implements IDeal {
   public dealManager: any;
 
   public primaryDao?: IDAO;
+  public swapTxHash?: string;
```

Comment:
```suggestion
  public swapTxHash: string | null = null;
```
Because below you set `      this.swapTxHash = this.dealDocument.swapTxHash || null;`